src/track/bpm.h

```diff
@@ -65,7 +65,7 @@ class Bpm final {
     };
 
     bool compareEq(
-            const Bpm& bpm,
+            Bpm bpm,
```

Comment:
Is getValue() not const? Why not?

Reply:
`getValue` and `compareEq` are both const. Is there some other function you are referring to?